node/Cargo.toml

```diff
@@ -37,67 +37,71 @@ futures = "0.3.14"
 jsonrpc-core = "15.1.0"
 
 # Substrate frames
-frame-benchmarking = { version = '3.0.0', git = 'https://github.com/paritytech/substrate.git', branch = "polkadot-v0.9.8" }
-frame-benchmarking-cli = { version = '3.0.0', git = 'https://github.com/paritytech/substrate.git', branch = "polkadot-v0.9.8" }
-try-runtime-cli = { version = '0.9.0', git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.8", optional = true }
+frame-benchmarking = { git = 'https://github.com/paritytech/substrate.git', branch = "polkadot-v0.9.9" }
+frame-benchmarking-cli = { git = 'https://github.com/paritytech/substrate.git', branch = "polkadot-v0.9.9" }
+try-runtime-cli = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.9", optional = true }
 
 # Substrate pallets rpc
-pallet-transaction-payment-rpc = { version = '3.0.0', git = 'https://github.com/paritytech/substrate.git', branch = "polkadot-v0.9.8" }
```

Comment:
So this is not considered substrate version 3 anymore ?

Reply:
Actually the version has been `4.0.0-dev`, it doesn't matter if we specify a commit or branch.